code/datums/station_traits/neutral_traits.dm

```diff
@@ -0,0 +1,23 @@
+/datum/station_trait/bananium_shipment
+	name = "Bananium Shipment"
+	trait_type = STATION_TRAIT_NEUTRAL
+	weight = 5
+	show_in_report = TRUE
```

Comment:
this shouldnt show in report imo, its funnier as a sekrit

Reply:
i'll think about it